### PR TITLE
Azure ARO deployment

### PR DESCRIPTION
**Pull request by petr-balogh**

### CODE REVIEW

**Comment by petr-balogh** [comment]
Scheduling verification job:
https://url.corp.redhat.com/91bd8e8

File: ocs_ci/utility/azure_utils.py [hunk 1]
```diff
@@ -461,6 +475,281 @@ def get_storage_account_properties(self, storage_account_name):
...
+            "aro_vnet_address_prefixes", "10.0.0.0/22"
+        )
+        master_subnet = config.ENV_DATA.get("aro_master_subnet", "master-subnet")
+        master_subnet_address_prefixes = config.ENV_DATA.get(
+            "aro_master_subnet_address_prefixes", "10.0.0.0/23"
+        )
+        worker_subnet = config.ENV_DATA.get("aro_worker_subnet", "worker-subnet")
+        worker_subnet_address_prefixes = config.ENV_DATA.get(
+            "aro_worker_subnet_address_prefixes", "10.0.2.0/23"
+        )
```
Comment (vavuthu) [inline ocs_ci/utility/azure_utils.py:615]: not a stopper.  could we make subnets as constants in defaults.py

**Review by vavuthu — APPROVED** [review]

**Comment by petr-balogh** [comment]
Running deployment verification here:
https://url.corp.redhat.com/f1204db

**Comment by petr-balogh** [comment]
New verification job:
https://url.corp.redhat.com/cecbea9

File: ocs_ci/utility/azure_utils.py (hunk 1 above)
Comment (petr-balogh) [reply ocs_ci/utility/azure_utils.py]: Done

**Comment by petr-balogh** [comment]
New verification job:
 https://url.corp.redhat.com/05ea34d

**Review by vavuthu — APPROVED** [review]

**Comment by petr-balogh** [comment]
New deployment verification after monitoring related error fixes:
https://url.corp.redhat.com/5b1baee

**Comment by petr-balogh** [comment]
New verification run triggered here after fixing a bug in monitoring change:
 https://url.corp.redhat.com/9a645ef

**Review by vavuthu — APPROVED** [review]

**Review by OdedViner — APPROVED** [review]

**Pull request merged**